build-ps/debian-zenfs/extra/apparmor-profile

```diff
@@ -28,16 +28,11 @@
   /etc/mysql/** r,
 
 # Allow pid, socket, socket lock file access
-  /var/run/mysqld/mysqld.pid rw,
-  /var/run/mysqld/mysqld.sock rw,
-  /var/run/mysqld/mysqld.sock.lock rw,
-  /var/run/mysqld/mysqlx.sock rw,
-  /var/run/mysqld/mysqlx.sock.lock rw,
-  /run/mysqld/mysqld.pid rw,
-  /run/mysqld/mysqld.sock rw,
-  /run/mysqld/mysqld.sock.lock rw,
-  /run/mysqld/mysqlx.sock rw,
-  /run/mysqld/mysqlx.sock.lock rw,
+  /{,var/}run/mysqld/mysqld.pid rw,
```

Comment:
I think we do not use this file. It is just a copy of upstream's packaging/deb-in/extra.
We maintain our own profile in debian//extra/apparmor.d/usr.sbin.mysqld.in.
Kind of a mess, and causing it easy to miss upstream changes to the AA profile. I don't know why we are not aligned here with the upstream.